src/app/item-review/item-review.component.ts

```diff
@@ -32,13 +34,27 @@ function ItemReviewController(
   vm.toggledFlags = [];
   vm.submitted = false;
   vm.isCollapsed = false;
+  vm.expandReview = false;
 
   vm.$onInit = () => {
     vm.canCreateReview = (vm.canReview && vm.item.owner);
-    vm.hasUserReview = ((vm.item.userRating) || (vm.item.userVote));
-    vm.expandReview = ((vm.item.isLocallyCached) && (vm.item.userVote !== 0));
-    if (!vm.item.mode) {
-      vm.item.mode = settings.reviewsModeSelection;
+
+    if (vm.item.destinyVersion === 1) {
```

Comment:
Use `vm.item.isDestinyOne()`. Then within the branch it's automatically a `D1Item`.